lua/nvim-tree/explorer/node.lua

```diff
@@ -14,13 +14,15 @@ local function get_dir_git_status(parent_ignored, status, absolute_path)
     return { file = "!!" }
   end
 
-  return {
-    file = status.files and status.files[absolute_path],
-    dir = status.dirs and {
-      direct = status.dirs.direct[absolute_path],
-      indirect = status.dirs.indirect[absolute_path],
-    },
-  }
+  if type(status) ~= nil and status ~= nil then
```

Comment:
Nitpick: this double check is unnecessary, a simple `if status then` should be enough

Reply:
Nitpicks wanted! First time writing anything in Lua (besides configs) and I wasn't sure if a variable was unset if its type was `nil` or .. yeah. Pushed a commit to fix.